joomla/joomla-cms
# [5.0] Unify the rendering of plugins, form fields and modules

### Summary of Changes
There are different ways for form fields, plugins and modules to render their layout files. This should be unified and all parts should use `FileLayout` to render the layouts.

### Testing Instructions
- Enable the stats plugin
- Enable the voting plugin
- Open the article form in the back end and create an article
- Set show voting to yes in the article options
- Make sure that the breadcrumbs plugin is enabled
- Open an article on the front

### Actual result BEFORE applying this Pull Request
Stats are displayed, the article form throws no error and the voting feature is displayed in the article details view.

### Expected result AFTER applying this Pull Request
Stats are displayed, the article form throws no error and the voting feature is displayed in the article details view.

### Link to documentations
Please select:
